### PR TITLE
TSPS-363 Make terralab cli executable

### PR DESCRIPTION
**Pull request by mmorgantaylor**
### Description 

With this PR, you can run `terralab [COMMAND]` from anywhere (once you've run `poetry install`), rather than needing to run `./terralab [COMMAND]` from the project directory.

Once this PR is merged and the new package is published to PyPi, I will test whether a `pip install terralab-cli` results in the same behavior.

### Jira Ticket
https://broadworkbench.atlassian.net/browse/TSPS-363

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -1,5 +1,6 @@
 [tool.poetry]
 name = "terralab-cli"
+# version is auto-incremented. do not edit.
```
Comment (jsotobroad) [inline pyproject.toml:3]: nice!

File: pyproject.toml [hunk 2]
```diff
@@ -8,9 +9,12 @@ packages = [
     {include = "terralab"}
 ]
 
+[tool.poetry.scripts]
+terralab = "terralab.cli:cli"
```
[the remark this answers is not in the document]
Comment (jsotobroad) [reply pyproject.toml:13]: oooo that's so simple lol

File: terralab/commands/auth_commands.py [hunk 3]
```diff
@@ -2,7 +2,7 @@
 
 import click
 
-from logic import auth_logic
+from terralab.logic import auth_logic
```
Comment (jsotobroad) [inline terralab/commands/auth_commands.py:5]: where these import changes related to stuff in this pr or from a previous pr?

File: terralab/terralab [hunk 4]
```diff
@@ -1,6 +0,0 @@
-#!/usr/bin/env poetry run python
```
Comment (jsotobroad) [inline terralab/terralab:1]: so when someone is developing, how do they make sure they can test on the latest code changes.  do they have to keep running `poetry install` so they can use `terralab` with the latest code or once they run `poetry install` once, `terralab` will always point to their latest code?

**Review by jsotobroad — COMMENTED** [review]
this is awesome, just had some clarifying questions

File: terralab/commands/auth_commands.py (hunk 3 above)
Comment (mmorgantaylor) [reply terralab/commands/auth_commands.py:5]: they were required to get the code working in the executable manner. i'm not sure why it wasn't an issue before and became an issue. this was inconsistently done previously.

File: terralab/terralab (hunk 4 above)
Comment (mmorgantaylor) [reply terralab/terralab:1]: nope, it's pretty magical, you can run `poetry install` the one time and then keep developing and when you run `terralab whatever` it uses the updated code. (i tested this when doing the updates to the version stuff that's also in this pr)

File: pyproject.toml (hunk 2 above)
[the remark this answers is not in the document]
Comment (mmorgantaylor) [reply pyproject.toml:13]: right? love it

File: terralab/terralab (hunk 4 above)
Comment (jsotobroad) [reply terralab/terralab:1]: thats super awesome

**Review by jsotobroad — APPROVED** [review]

**Pull request merged**